### PR TITLE
New package: xye-0.12.2

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

File: srcpkgs/xye/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'xye'
+pkgname=xye
+version=0.12.2
+revision=1
+build_style=gnu-configure
+makedepends="SDL-devel SDL_image-devel SDL_ttf-devel"
```
Comment (Vaelatern) [inline srcpkgs/xye/template]: `SDL-devel` is redundant.

**Comment by asiekierka** [comment]
Fixed.

File: srcpkgs/xye/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="SDL_image-devel SDL_ttf-devel"
+short_desc="Puzzle game inspired by Kye"
+maintainer="Adrian Siekierka <asiekierka@gmail.com>"
+license="zlib, custom"
+homepage="http://xye.sourceforge.net"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum=5d7c04f37d6b4a1821594512c7ddbc8189f968e08ecfd5af55231302f2f917d8
+
+post_install() {
+	vdoc ReadMe.html
```
Comment (Vaelatern) [inline srcpkgs/xye/template]: Okay, last change before I'm ready to merge.
```
diff --git a/srcpkgs/xye/template b/srcpkgs/xye/template
index cd6ecea..96ff65a 100644
--- a/srcpkgs/xye/template
+++ b/srcpkgs/xye/template
@@ -13,6 +13,9 @@ checksum=5d7c04f37d6b4a1821594512c7ddbc8189f968e08ecfd5af55231302f2f917d8
 
 post_install() {
        vdoc ReadMe.html
+       vdoc ChangeLog
+       vdoc AUTHORS
+       vdoc NEWS
        vlicense COPYING
        vlicense res/detailed_COPYING
 }
```

Comment (asiekierka) [inline srcpkgs/xye/template]: And done!

**Pull request merged**